pip.md

```diff
@@ -75,12 +80,12 @@ RAPIDS users can once again install RAPIDS via pip!  This is an **experimental r
 	pip install cuml-cu11 --extra-index-url=https://pypi.ngc.nvidia.com
 	pip install cugraph-cu11 --extra-index-url=https://pypi.ngc.nvidia.com
 
-> <i class="fas fa-info-circle text-white"></i> The RAPIDS pip packages are hosted on the NVIDIA NGC index today.
-
-> <i class="fas fa-info-circle text-white"></i> On ARM architecture (aarch64), cupy needs to be installed separately:
+> <i class="fas fa-info-circle text-white"></i> The RAPIDS pip packages are hosted on the NVIDIA NGC index today.  <br>
+> <i class="fas fa-info-circle text-white"></i> On ARM SBSA, install cupy separately using this command:
```

Comment:
Does it matter if the user does this command before or after the other installs? Mostly curious if the versioning in our wheels would override the cupy-cuda11x or if it would remain.

Reply:
It must be installed after RAPIDS, to the best of my knowledge.

Reply:
Got it - maybe we change the wording from

> On ARM SBSA, install cupy separately using this command:

To something like:

> On ARM SBSA, install cupy after RAPIDS libraries using this command: